lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -45,11 +46,15 @@ def ask_question_loop
     end
 
     def activate
+      if key_exist? && force
+        backup_key
+      end
+
```

Comment:
What happened if we have good `v2_key.backup` and bad `v2_key`and fetching would fail ? - we would loose good backup.  I think flow should not alter existing back-up if fetching failing.